radio/src/gui/colorlcd/radio_hardware.cpp

```diff
@@ -278,7 +278,16 @@ void RadioHardwarePage::build(FormWindow * window)
 
 #if defined(AUX_SERIAL)
   new StaticText(window, grid.getLabelSlot(), STR_AUX_SERIAL_MODE);
+//OW
+#if defined(TELEMETRY_MAVLINK) && !defined(CLI) && !defined(DEBUG)
```

Comment:
Have you checked what happens when `TELEMETRY_MAVLINK && (CLI || DEBUG)` are defined? Also, on the TX16S, you can define all these, and use them in parallel, as you have 2 AUX serials.